dashboard/app/views/scripts/show.html.haml

```diff
@@ -1,6 +1,8 @@
 -# This variable will get any data we need to pass along to scripts/show.js
 - script_data = @script.summarize(true, current_user)
-- additional_script_data = {course_name: @script.course&.name, show_redirect_warning: @show_redirect_warning, redirect_script_url: @redirect_script_url, section: @section, user_type: current_user&.user_type, user_id: current_user&.id, is_verified_teacher: current_user&.authorized_teacher?, locale: Script.locale_english_name_map[request.locale]}
+- sections = @current_user.try{|u| u.sections.where(hidden: false).select(:id, :name, :script_id)}
+- sections = sections.map { |section| section.attributes.merge!({"isAssigned" => section[:script_id] == @script.id})}
```

Comment:
apologies for not commenting on this earlier, but the best practice is for ruby code to go in the controller method: https://github.com/code-dot-org/code-dot-org/blob/0b2856ab5d99d3fa1e6e1ecc7277957f9ec18358/dashboard/app/controllers/scripts_controller.rb#L12

one benefit is that our linting rules will run on it, catching potential errors. 

In this case, please at least move the definition of `sections` to the controller method, name it `@sections` and then access it below as `@sections`.

ideally, all of `scriptOverviewData` would move to the controller method but it's ok if you don't want to tackle that now.

Reply:
done. updated here and for courses.